src/app_service/service/community/service.nim

```diff
@@ -86,6 +86,62 @@ QtObject:
     result.allCommunities = initTable[string, CommunityDto]()
     result.myCommunityRequests = @[]
 
+  proc findChatById(id: string, chats: seq[ChatDto]): ChatDto =
+    for chat in chats:
+      if(chat.id == id):
+        return chat
+
+  proc findById(id: string, chats: seq[Chat]): int =
```

Comment:
Maybe `findIndexById` would be clearer a name

Reply:
we are returning a chat Item not the index. or do you mean a function that returns index and then I get it by calling [index] ?

Reply:
The function above indeed returns the ChatItem, but this one only returns the index since it returns an `int`